spec/jobs/replace_github_non_permalinks_spec.rb

```diff
@@ -60,4 +60,34 @@
       expect(post.raw).to eq(updated_post)
     end
   end
+
+  describe "#excluded?" do
+    before do
+      SiteSetting.github_permalinks_exclude = "README.md|discourse/discourse/directory/file.rb|discourse/onebox/docs/*|discourse/anotherRepo/*|someUser/*"
+    end
+
+    it 'returns true when it should be excluded'  do
+      result1 = Jobs::ReplaceGithubNonPermalinks.new.excluded?("discourse", "discourse", "README.md")
```

Comment:
If you're not re-using `resultX` here I don't think it makes sense to put them in variables.

One thing you could do to shorten it is create a `let(:job) { described_class.new }` above your test code, then your tests could be shorter:

`expect(job.excluded?("discourse", "discourse", "directory/file.rb")).to eq(true)`
...